libtrip-notification/src/main/java/com/mapbox/navigation/trip/notification/MapboxTripNotification.kt

```diff
@@ -55,8 +57,10 @@ import kotlinx.coroutines.channels.ClosedSendChannelException
  * Default implementation of [TripNotification] interface
  *
  * @param applicationContext is [Context]
```

Comment:
```suggestion
 * @param applicationContext the application's [Context]
```

Reply:
Make the same change for the other two places where it says`* @param applicationContext is [Context]` in the `MapboxTripNotification.kt` file